### ISSUE TITLE
Course Scan ID heading is an <h5> but should be an <h2>

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/canvas-app-explorer#656]
Based on the [Accessibility Review Report](https://docs.google.com/spreadsheets/d/1vt4uJuSq3itNu8Ouq6KI-4KOdOlOi_T9PoqOVy0e3T0):

Severity: 2 - Minor

Test: Alt Text helper Landing Page

Issue notes:
Course Scan ID heading is a h5 element but should be a h2 element

Category: 
Semantics and Structure

Criterion:
[1.3.1 – Info and Relationships](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#info-and-relationships)

Impact:
People using any type of assistive technology will have barriers or difficulty navigating or accessing this information.

Most affects people who are/have:
Blind, Low vision, Neurodivergent

**Comment by jonespm** [comment]
Related to this it looks like Start Review is an h6 and _probably_ should also be an h2. I'm not sure why those heading levels were chosen here. It looks like it was done a long time ago.
